dotnet/src/Functions/Functions.OpenApi/OpenApiKernelPluginFactory.cs

```diff
@@ -207,44 +208,16 @@ async Task<RestApiOperationResponse> ExecuteAsync(Kernel kernel, KernelFunction
         {
             try
             {
-                // Extract function arguments from context
-                var arguments = new KernelArguments();
-                foreach (var parameter in restOperationParameters)
-                {
-                    // A try to resolve argument by alternative parameter name
-                    if (!string.IsNullOrEmpty(parameter.AlternativeName) &&
-                        variables.TryGetValue(parameter.AlternativeName!, out object? value) &&
-                        value is not null)
-                    {
-                        arguments.Add(parameter.Name, value);
-                        continue;
-                    }
-
-                    // A try to resolve argument by original parameter name
-                    if (variables.TryGetValue(parameter.Name, out value) &&
-                        value is not null)
-                    {
-                        arguments.Add(parameter.Name, value);
-                        continue;
-                    }
-
-                    if (parameter.IsRequired)
-                    {
-                        throw new KeyNotFoundException(
-                            $"No variable found in context to use as an argument for the '{parameter.Name}' parameter of the '{pluginName}.{operation.Id}' Rest function.");
-                    }
-                }
```

Comment:
What happens to this logic?

Reply:
It's not needed since the RestApiRunner does the arguments lookup when building sever, path, query string and headers.